docs/examples/simulation.md

```diff
@@ -0,0 +1,382 @@
+    name = "Bulk simulation for measurement",
+    type = "MD",
+    software_configuration = [python_config, openmm_config],
+    condition = [
+        cript.Condition(key="time_duration", type="value", value=50.0, unit="ns"),
+        cript.Condition(key="temperature", type="value", value=450.0, unit="K"),
+        cript.Condition(key="pressure", type="value", value=1.0, unit="bar"),
+        cript.Condition(key="number", type="value", value=31),
+    ],
+    prerequisite_computation = equi,
+)
+
+ana = cript.Computation(
+        name = "Density analysis",
+        type = "analysis",
+        software_configuration = [python_config],
+        prerequisite_computation = bulk,
+)
+
+# Add all these computations to the experiment.
+experiment.computation += [init, equi, bulk, ana]
+```
+
+
+!!! note "Computation types"
+    The allowed `Computation` types are listed under <a href="https://criptapp.org/keys/computation-type/" target="_blank">computation types</a> in the CRIPT controlled vocabulary.
+
+!!! note "Condition keys"
+    The allowed `Condition` keys are listed under <a href="https://criptapp.org/keys/condition-key/" target="_blank">condition keys</a> in the CRIPT controlled vocabulary.
+
+
+# Create and Upload Files
+
+New we'd like to upload files associated with our simulation. First, we'll instantiate our File nodes under a specific project.
+
+```python
+packing_file = cript.File("Initial simulation box snapshot with roughly packed molecules", type="snapshot", source="path/to/local/file")
+forcefield_file = cript.File(name="Forcefield definition file", type="data", source="path/to/local/file")
+snap_file = cript.File("Bulk measurement initial system snap shot", type="snapshot", source="path/to/local/file")
+final_file = cript.File("Final snapshot of the system at the end the simulations", type="snapshot", source="path/to/local/file")
+```
+
+!!! note
+The `source` field should point to any file on your local filesystem.
+
+!!! info
+Depending on the file size, there could be a delay while the checksum is generated.
+
+Next, we'll upload the local files by saving the File nodes.
+
+```python
+# @ Navid, add code of how to upload the files.
+```
+
+# Create Data
+
+Next, we'll create a <a href="../../nodes/data" target="_blank">`Data`</a> node which helps organize our <a href="../../nodes/file" target="_blank">`File`</a> nodes and links back to our <a href="../../nodes/computation" target="_blank">`Computation`</a> objects.
+
+```python
+packing_data = cript.Data(
+    name = "Loosely packed chains",
+    type = "computation_config",
+    file = [packing_file],
+    computations = [init],
+    notes = "PDB file without topology describing an initial system.",
+)
+
+forcefield_data = cript.Data(
+    name = "OpenMM forcefield",
+    type = "computation_forcefield",
+    file = [forcefield_file],
+    computation = [init],
+    notes = "Full forcefield definition and topology.",
+)
+
+equi_snap = cript.Data.create(
```

Comment:
do we have a `.create()` method on the data node? was this mistakenly carried over from the old SDK docs?

our version of this code walk through I think is here: https://github.com/C-Accel-CRIPT/Python-SDK/blob/develop/docs/examples/synthesis.md

`docs/examples/synthesis.md`

Reply:
Yes, that is still from the old documentation.

I couldn't finish the transformation, because of the File node issues discussed in #175

Reply:
no, that we merged #175 I finished it!